assets/js/playlist.js

```diff
@@ -59,6 +62,37 @@
       url: $(this).attr('url'),
       data: $(this).serialize()
     }).done(function(response){
-      $('#playlists').html(response)
+      $('#playlists').html(response);
     });
-  });
\ No newline at end of file
+  });
+
+  // ajax for deleting songs
+  $(document).on('submit', '.deleteSong', function( event ){
+    event.preventDefault();
+    $.ajax({
+      method: 'POST',
+      url: $(this).attr('action'),
+      data: $(this).serialize()
+    }).done(function(response){
+      //remove song from the template
+      $("#"+response.song_id).parent( "li" ).remove()
+      song_ids.pop();
+      console.log(song_ids);
```

Comment:
remove console log. make sure to remove it everytime you commit.

Reply:
Updated.